client/components/reasonable-demo/reasonableDemo.re

```diff
@@ -0,0 +1,43 @@
+open Js;
+
+let component = ReasonReact.statelessComponent "ReasonableDemo";
+
+type actions =
+  | Counter_Increment
+  | Counter_Decrement
+  | Counter_Add
+  [@@bs.deriving {accessors: accessors}];
+
+let make ::name ::count ::onClick _children => {
+  let text = ReasonReact.stringToElement;
+  let clickIt _event _state _self => onClick () [@bs];
+  {
+    ...component,
+    render: fun () self =>
```

Comment:
Small note: we've released a new update that turns this into `render: fun self =>`. No need for the `()` anymore. If you change `reason-react` to `^0.2.3` then it'll Just Work.